cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2193,6 +2193,14 @@
   </property>
 
   <property>
+    <name>mapreduce.task.metrics.enabled</name>
+    <value>true</value>
+    <description>
+      Determines if task level metrics have to emitted for map-reduce programs.
```

Comment:
nit: `Determines if task level metrics have to be emitted for map-reduce programs.?`

Reply:
Why a specific config for mapreduce?? Shouldn't it be a system arguments just like others, which can be scoped at different level? E.g. in runtime arguments

`system.metrics.enabled = false` - disable metrics for all containers
`task.mapper.system.metrics.enabled = false` - disable metrics for all mappers
`task.*.system.metrics.enabled = false` - disable metrics for all mappers and reducers, but not driver

The same applies to any program type, not really specific for MR.

Reply:
currently we do support overriding this value as runtime argument for the program, do you suggest not have it as configurable in `cdap-site` but only in runtime arguments ?

Reply:
I mean, why it is `mapreduce` specific?